core/ecschema-metadata/src/Metadata/Class.ts

```diff
@@ -133,7 +133,8 @@ export abstract class ECClass extends SchemaItem implements CustomAttributeConta
 
   /**
    * Searches, case-insensitive, for a local ECProperty with the name provided.
```

Comment:
'local ECProperty' implies one defined on this class and not one inherited ... but the default is to get the property local or inherited.